docs/topic-guides/blendable-web-apps/uniform-design-system.md

```diff
@@ -0,0 +1,90 @@
+# Uniform Design System
+
+## Introduction
+
+Uniform design system is a user interface toolkit created by Starcounter. It gives a common look and feel to blendable apps. It is a link between app authors and solution owners.
+
+Unlike many design systems, Uniform is open-ended and themeable. It can be configured, extended or replaced in parts to achieve anything that is possible with HTML, CSS, and JavaScript.
+
+## Mental model of client-side rendering by the Starcounter host
+
+Starcounter host follows a layered design that separates the responsibilities of the app author, the design system and the solution owner. Uniform fits in this design as the middle layer. 
+
+The full design consists of:
+
+1. Native HTML5 and Shadow DOM
+2. Uniform design system
+3. Theme configuration through CSS custom properties
+4. Extend or replace the theme with HTML, CSS, and JavaScript
+
+### Native HTML5 and Shadow DOM
+
+As an app author, you provide the views for your apps with a clear separation of the content and the presentation.
+
+In the content part, you provide the UI components that should be rendered by the Starcounter host as unstyled native HTML5 elements. 
```

Comment:
Can't I use HTML4 elements? or some upcoming 5.4? I think the version does not matter here at all.

Also, as an application author, I'm not interested in any technology that renders my UI unstyled. 
"you provide the UI components that should be rendered by the Starcounter host as unstyled ... elements" gives me negative impression.